### PR TITLE
Update install requirements & Rewrite emotes

### PR DESCRIPTION
**Pull request by avengerweb**
- Mcrypt is deprecated form php7.1 and removed in laravel 5.1 (We using 5.2)
- Add information about debugging
- Rewrite emotes

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -106,6 +81,28 @@ From now on you can perform HTTP requests to the API at **http://localhost/TC-JS
...
+
+`APP_DEBUG=true`
+
+Now you can see error messages in your browser & logs too.
+
+**Be careful! Do not use debug mode on production server.**
+
+## Developing environment
+
+You really doesn't need install apache or nginx on your local PC for developing, that easily to use integrated web server.
```
Comment (FrancescoBorzi) [inline INSTALL.md]: you really **don't** need

Comment (avengerweb) [inline INSTALL.md]: Ok, thx :)

**Comment by FrancescoBorzi** [comment]
another thing, I guess the full version still requires the MCRYPT, doesn't it ?

**Comment by avengerweb** [comment]
I will reopen it, bad author name :)
Full version of what? In laravel 5.1 mcrypt dependency was replaced with OpenSSL.

**Comment by FrancescoBorzi** [comment]
latest full version of the TC API https://github.com/ShinDarth/TC-JSON-API/releases

**Comment by avengerweb** [comment]
And? So, readme included in this realease. and install guid, but now I think, we need to make new release.

**Comment by avengerweb** [comment]
mcrypt isn`t need for new version, too. And new release would be after we switched to Laravel 5.2.

**Comment by FrancescoBorzi** [comment]
yes I agree we need to release a new version, but I don't have the time to perform enough tests right now.. could you do that? making sure everything works with the latest changes

**Comment by avengerweb** [comment]
Ok, I will try do that, but in another pull request with version change, I think that would be right.

**Comment by FrancescoBorzi** [comment]
ok, let me know and thanks a lot for your cooperation ;)

**Comment by avengerweb** [comment]
Are you merge this PR?

**Comment by FrancescoBorzi** [comment]
yes but I wouldn't edit the README.md until we also release the new full version, in order to make the installation instructions consistent

**Comment by avengerweb** [comment]
I checked all what I can. Now you can merge, I think.

**Comment by FrancescoBorzi** [comment]
merged, thanks a lot!

**Pull request merged**